account-service/src/main/resources/application.properties

```diff
@@ -1,6 +1,6 @@
 spring.application.name=account-service
 server.port=8020
-spring.data.mongodb.host=localhost
-spring.data.mongodb.port=27017
-spring.data.mongodb.database=AccountServiceUsers
+#spring.data.mongodb.host=localhost
```

Comment:
https://github.com/we-3hackathon/bdonor/blob/e81f5b4d18581c56ab3e031675033d7a09cb62ef/account-service/src/main/resources/application.properties#L3-L5